### PR TITLE
#5408: Add test_perf file for ttnn stable diffusion model

### PR DESCRIPTION
**Pull request by vigneshkeerthivasanx**
This PR contains test_perf file for profiling the inference, compile and throughput time for ttnn stable diffusion model.

### CODE REVIEW

File: tests/ttnn/integration_tests/stable_diffusion/test_perf_stable_diffusion.py [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+@pytest.mark.parametrize(
+    "expected_inference_time, expected_compile_time",
+    (
+        (
+            140,
+            140,
+        ),
+    ),
+)
+def test_perf(device, expected_inference_time, expected_compile_time, model_name, reset_seeds):
```
Comment (eyonland) [inline tests/ttnn/integration_tests/stable_diffusion/test_perf_stable_diffusion.py:66]: Can we please kick this test off from [https://github.com/tenstorrent-metal/tt-metal/blob/main/tests/scripts/run_performance.sh](https://github.com/tenstorrent-metal/tt-metal/blob/main/tests/scripts/run_performance.sh) ?

**Review by eyonland — CHANGES_REQUESTED** [review]

File: tests/ttnn/integration_tests/stable_diffusion/test_perf_stable_diffusion.py [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+    return noise_pred
+
+
+def latent_expansion(latents, scheduler, t):
+    latent_model_input = torch.cat([latents] * 2)
+    latent_model_input = scheduler.scale_model_input(latent_model_input, timestep=t)
+    return latent_model_input
+
+
+@skip_for_wormhole_b0()
```
[the remark this answers is not in the document]
Comment (vigneshkeerthivasanx) [reply tests/ttnn/integration_tests/stable_diffusion/test_perf_stable_diffusion.py]: > Does this test fail on WH?

We are facing indefinite hang issue while testing test_performance on WH. We are facing hang issue at different instance of test_performance while testing each time.

File: tests/ttnn/integration_tests/stable_diffusion/test_perf_stable_diffusion.py (hunk 1 above)
Comment (vigneshkeerthivasanx) [reply tests/ttnn/integration_tests/stable_diffusion/test_perf_stable_diffusion.py]: > Can we please kick this test off from https://github.com/tenstorrent-metal/tt-metal/blob/main/tests/scripts/run_performance.sh ?

Yes. It has been updated in run_performance.sh

**Comment by tt-rkim** [comment]
Please run the performance pipeline and paste the result here.

**Review by arakhmati — APPROVED** [review]

**Comment by vigneshkeerthivasanx** [comment]
Passing CI Links:
All post-commit tests : https://github.com/tenstorrent-metal/tt-metal/actions/runs/8150056783

**Review by tt-rkim — APPROVED** [review]

**Pull request closed without merging**